app/test_main.py

```diff
@@ -1,3 +1,25 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (100, 100, [21, 17]),
+        (-1, -1, [0, 0]),
+    ],
+    ids=[
+        "0 cat/dog years should convert into 0 human age.",
+        "15 cat/dog years should convert into 1 human age.",
+        "23 cat/dog years should convert into 1 human age.",
+        "24 cat/dog years should convert into 2 human age.",
+        "100 cat/dog years should convert into 21/17 human age.",
+        "Negative values should return 0 human age.",
+    ]
+)
+def test_get_human_age(cat_age: int, dog_age: int, expected: int) -> None:
```

Comment:
The expected parameter is defined as an int, but it is actually a list. You should change the type hint for 'expected' to 'list' to match the actual data type being used.